### DISCUSSION TITLE
is this library is usable for Adafruit Round RGB TTL TFT Display - 4" 720x720 - No Touchscreen - NV3052C

### DISCUSSION THREAD

**Discussion by gellary** [moononournation/Arduino_GFX#515] [Q&A]
hi ~i m new in project display,  is this library works in arduino ide for NV3052C driver ?
and what should i aware  ? or any example code for  Adafruit Round RGB 4" TTL TFT Display?

**Comment by moononournation** [comment]
I did not have this display in hand, so not tested.

**Comment by jfseaman** [comment]
Yes. I use it for that display

Board: Adafruit Qualia ESP32-S3 RGB666  https://www.adafruit.com/product/5800
Display: Adafruit Round RGB TTL TFT Display - 4" 720x720 - No Touchscreen - NV3052C https://www.adafruit.com/product/5793

see my repo at https://github.com/jfseaman/Qualia_ESP32S3_RGB666
